webpack.config.js

```diff
@@ -62,6 +68,12 @@ module.exports = {
 							dev
 						}
 					}
+				},
+				{
+					test: /\.(png|svg|jpg|gif)$/,
```

Comment:
We should likely extract this regex rather than duplicate it

Reply:
done - I extracted a bit more than that though